README.md

```diff
@@ -57,6 +57,8 @@ Currently, this project supports the following environments.
 | Linux   | 1.21       | arm64        |
 | MacOS   | 1.22       | amd64        |
 | MacOS   | 1.21       | amd64        |
+| MacOS   | 1.22       | arm64        |
```

Comment:
How about saying `Mac OS 13` for the AMD entries?

Reply:
PTAL https://github.com/open-telemetry/opentelemetry-go/pull/5577/commits/4c17a876ce392d773f8d53740d06b780afaf49d1